packages/oui-progress/_mixins.less

```diff
@@ -105,6 +104,7 @@
     font-size: @oui-progress-bar-text-font-size;
     font-weight: @oui-progress-bar-text-font-weight;
     line-height: @oui-progress-height;
+    white-space: nowrap;
```

Comment:
Why force nowrap ?
You can have a 2 or 3 words on a label, with a break line and without ellipsis